en/docs/Learn/RateLimiting/blacklisting-whitelisting.md

```diff
@@ -71,27 +81,37 @@ To blacklist a request,
 
 1.  Log in to the Admin Portal using the URL `https://localhost:9443/admin` and your admin credentials.
 2.  Click **Black List** under the **Throttle Policies** section and click **Add Item** .
-    ![]({{base_path}}/assets/attachments/103335009/103335010.png)
+
+    ![](../../assets/img/Learn/add-blacklist-policy.png)
+
 Select the item to black list, enter a value and click **Blacklist** .
-![]({{base_path}}/assets/attachments/103335009/103335024.png)
 
-!!! warning
-Note that you have to use " **/** " always infront of the **${context}** value when blacklisting the APIs with API context. E.g. **/test/1.0.0.** The sample provided in the product does not include "/" due to a known issue.
-!!! info
-You can temporary on/off the blacklisting condition by enabling/disabling the **Condition status** that is auto enabled when a blacklisting condition is created.
-![]({{base_path}}/assets/attachments/103335009/103335013.png){height="250"}
+![](../../assets/img/Learn/select-blacklist-condition.png)
+
+<div class="admonition info">
+    <p class="admonition-title">Note</p>
+<p>You can temporary on/off the blacklisting condition by enabling/disabling the <b>Condition status</b> that is auto enabled when a blacklisting condition is created. </p>
+</div>
+
+![](../../assets/img/Learn/blacklist-condition-status.png)
 
 ##### Blacklisting PhoneVerification API
 
-As described above you can blacklist requests for APIs, by Applications, to IP Addresses and for Users. Let's see how we can blacklist the requests come to the [PhoneVerification API](https://docs.wso2.com/display/AM2xx/Quick+Start+Guide#QuickStartGuide-PublishingtheAPI) that we published in Quick Start Guide.
+As described above you can blacklist requests for APIs, by Applications, to IP Addresses and for Users. Let's see how we can blacklist the requests for an API.
 
 1.  Log in to the Admin Portal using the URL `https://localhost:9443/admin` and your admin credentials.
 2.  Click **Black List** under the **Throttle Policies** section and click **Add Item** .
 3.  Select **API Context** and provide the Context of PhoneVerification API with version as the **Value.**
-    ![]({{base_path}}/assets/attachments/103335009/103335014.png)4.  Click **Blacklist.**
-5.  Now login to API Store using the URL `https://localhost:9443/s                      tore` and [invoke the API](https://docs.wso2.com/display/AM2xx/Quick+Start+Guide#QuickStartGuide-InvokingtheAPI) .
+
+    ![](../../assets/img/Learn/select-context-blacklist.png)
+
+4.  Click **Blacklist.**
+5.  Now login to API Store using the URL `https://localhost:9443/devportal` and invoke the API.
```

Comment:
Shall we change API Store to Developer Portal

Reply:
@AmaliMatharaarachchi Fixed. Thanks.